snmp/tests/test_e2e_core_vs_python.py

```diff
@@ -452,11 +452,6 @@ def test_e2e_profile_netapp(dd_agent_check):
     assert_python_vs_core(dd_agent_check, config)
 
 
-def test_e2e_profile_palo_alto(dd_agent_check):
-    config = common.generate_container_profile_config('palo-alto')
-    assert_python_vs_core(dd_agent_check, config)
-
-
```

Comment:
Why delete this test ?

Reply:
That's because python integration doesn't support the feature to "extend profile with the same name".

Hence, the profile is skipped in Python (more precise, it fails with `RecursionError`, see change in python integration).

Remove this test shouldn't cause lost of coverage since most/all the syntaxes are also tested in other tests in `test_e2e_core_vs_python.py`